dinh_ngoc_do/Action_ViewDB.php

```diff
@@ -0,0 +1,47 @@
+<?php 
+include '../MySQL/connect.php';
+
+//Delete User
+
+if (isset($_GET['deleteuser'])) {
+    $id = $_GET['deleteuser'];
+    $sql = "DELETE FROM do_test1 WHERE id = '$id'";
+    $result = $conn->query($sql);
+    
+    if ($result) {
+        echo "<script> alert('Delete Successfuly'); </script>";
+        header('Location:http://localhost/Lession1/MySQL/View_DB.php');
+    } else {
+        echo "<script> alert('Delete Failed'); </script>";
+    }
+}
+
+//Delete All User
+
+if (isset($_GET['deletealluser'])) {
+    $sql = "DELETE FROM do_test1";
+    $result = $conn->query($sql);
+
+    if ($result) {
```

Comment:
a cũng chưa thấy lỗi gì ở đoạn này, thử đặt debug vào đây xem result đang trả lại là gì?
ko được thì thay query `truncate do_test1` xem sao

Reply:
![image](https://user-images.githubusercontent.com/83223275/129534587-3683071c-f92d-4c01-907e-76134d596825.png)
dạ vâng e đổi sreach thành get thì hiện tại nó lấy đc giá trị trong ô input của e rồi nhưng mà nó vẫn chưa vào đc cái if của em , vì e tính là bao h nó phải vào đc cái alert Successfuly của em cơ thì lúc đấy e mới làm tiếp cái bảng in giá trị ra

Reply:
e thấy 2 lỗi của sreach với delete giống nhau , còn debug thì trước anh có dạy nhưng mà lâu quá nên em quên mất rồi :((( tối nay a chữa lại giúp em với

Reply:
ok e